index.html

```diff
@@ -20,7 +20,7 @@
         <div class="classlinks">
 
             <header>
-                <img src="img/dayton-logo.png" class="gdi-dayton-logo" alt="GDI Dayton">
+                <img src="img/gdi-logo.png" class="gdi-dayton-logo" alt="GDI Seattle">
```

Comment:
not a huge thing, but do we want to change class names to ours for sake of being consistent?